expandedTree/expandedTree.go

```diff
@@ -36,24 +36,35 @@ func ReadTree(newickFilename string, fastasFilename string) *ETree {
 
 //read in tree from filename
 func ReadNewick(filename string) (*ETree, error) {
-	var line string
+	var singleLineTree string
```

Comment:
Just an FYI that you can put this all on one line if you like:
var singleLineTree string = ReadMultiLineTree(filename)